src/renderer.ts

```diff
@@ -651,7 +1153,10 @@ export function matchColumns(node: SVGGElement | HTMLElement, columns: {column:
  * renders a stacked column using composite pattern
  */
 class StackCellRenderer implements ICellRendererFactory {
-  constructor(private readonly nestingPossible: boolean = true) {
+  private readonly nestingPossible: boolean; //Readonly inside constructor was not working for me.
```

Comment:
I am using 2.1.1 but was throwing me error when I keep the master branch version